tests/testAutopas/tests/autopasInterface/Newton3OnOffTest.h

```diff
@@ -56,4 +56,17 @@ class Newton3OnOffTest : public AutoPasTestBase,
    */
   template <bool useNewton3, class Container, class Traversal>
   std::pair<size_t, size_t> eval(autopas::DataLayoutOption dataLayout, Container &container, Traversal traversalOption);
+
+  struct PrintToStringParamName {
+    template <class ParamType>
+    std::string operator()(const testing::TestParamInfo<ParamType> &info) const {
```

Comment:
only in this function you will need the toString() functions of the option objects.